src/CosmosDB/CosmosDB/help/Get-AzManagedCassandraDatacenter.md

```diff
@@ -0,0 +1,119 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.CosmosDB.dll-Help.xml
+Module Name: Az.CosmosDB
+online version: https://docs.microsoft.com/en-us/powershell/module/az.cosmosdb/get-azmanagedcassandradatacenter
+schema: 2.0.0
+---
+
+# Get-AzManagedCassandraDatacenter
+
+## SYNOPSIS
+Gets the ManagedCassandra Datacenters.
+
+## SYNTAX
+
+```
+Get-AzManagedCassandraDatacenter -ResourceGroupName <String> -ClusterName <String> [-DataCenterName <String>]
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+This cmdlet can be used to list the ManagedCassandra Datacenters in a Cluster and get a datacenter by its Name.
+
+## EXAMPLES
+
+### Example 1: Get a Managed Cassandra Datacenter by Name
+```powershell
+PS C:\> Get-AzManagedCassandraDatacenter -ResourceGroupName "test-powershell" -ClusterName "cluster1" -DataCenterName "dc1"
+
+Id                                                                                                                                                           Name Properties
+--                                                                                                                                                           ---- ----------
+/subscriptions/{subscriptionId}/resourceGroups/test-powershell/providers/Microsoft.DocumentDB/cassandraClusters/cluster1/dataCenters/dc1 dc1  Microsoft.Azure.Commands.CosmosDB.Models.PSDataCenterResourceProperties
+```
```

Comment:
You see the default output format is hardly readable -- Id is too long that it takes over the whole screen, yet it's not at all important.
Please consider how you want to display the output, the important ones should be put first, the less important ones should be after or hidden.
Please refer to https://github.com/Azure/azure-powershell/blob/main/documentation/development-docs/generate-format.ps1xml-file.md for how to update the output format